packages/common/src/intl/locales/en.json

```diff
@@ -15,5 +15,6 @@
   "app.transactions": "Transactions",
   "app.welcome": "Hello, {name}!",
   "breadcrumb.item": "#{id}",
-  "button.ok": "ok"
-}
+  "button.ok": "ok",
+  "button.open-the-menu": "Open the menu"

```

Comment:
I realise i'm unsure of the convention with the keys here

Reply:
I should write up where I've gone with it.. I don't really know what's best and haven't given it enough thought yet.